tests/motd/test_transformers.py

```diff
@@ -71,8 +71,8 @@ def test_correct_output(self, result: Callable[["str | dict", bool], str], sourc
             "<span style='color:rgb(170, 0, 0);text-shadow:0 0 1px rgb(42, 0, 0)'>8</span>"
             "<span style='color:rgb(170, 0, 170);text-shadow:0 0 1px rgb(42, 0, 42)'>9</span>"
             "<span style='color:rgb(255, 170, 0);text-shadow:0 0 1px rgb"
-            + str((64, 42, 0) if bedrock else (42, 42, 0))  # noqa: W503
-            + "'>10</span>"  # noqa: W503
+            + str((64, 42, 0) if bedrock else (42, 42, 0))
+            + "'>10</span>"
```

Comment:
Ruff doesn't enforce W503 at all, as it's actually against the PEP 8 suggested formatting rules, this is something that was only imposed by flake8 as something they considered to be a better practice. In fact W503 clashes with black, so we wouldn't want to be enforcing it anyway.